ng2-components/config/assets/tslint.json

```diff
@@ -59,7 +59,7 @@
         "no-unused-variable": true,
         "no-use-before-declare": true,
         "no-var-keyword": true,
-        "no-var-requires": false,
+        "no-var-requires": true,
```

Comment:
this raises lots of errors for TypeScript code where we use require to resolve images, also the vendor/polyfill files